policyengine_us/variables/gov/states/az/tax/income/credits/family_tax_credit/az_family_tax_credit.py

```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class az_family_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Family Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
```

Comment:
and remove from formula
```suggestion
    defined_for = "az_family_tax_credit_eligible"
```